lib/index.js

```diff
@@ -455,11 +358,12 @@ internals.diffTables = (migrationGroup, joinTables, allTables, mode, cb) => {
             tableNext();
         });
     },
-    () => {
+    (ignoreError) => {
 
-        // there can't be any actual errors
+        // There can't be any actual errors because here,
+        // an error gives us information (That the table isn't there)
```

Comment:
I don't understand!  It looks like the error is passed along.  How is this any different than,
```js
    (err) => {
        if (err) return cb(err);
        return cb(null, knexTableDeltaObject);
    }
```